src/Movie.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+const Movie = ({ id, poster, title, rating, releaseDate, backdrop }) => {
+  return (
+    <section id={id}>
+      <article>
+        <img src={poster} alt={"Poster for " + title}/>
```

Comment:
Since you are visually displaying the title right below the image, you may not need alt text! Remember its redundant for the screen reader to say the same thing twice. If you decide to do that, make sure you have an empty alt tag so its not flagged `alt=""`

I was curious how Rotten Tomatoes does their alt text, and they got an 83 on Lighthouse because they don't have any alt attributes at all :(